src/css/profile/mobile/common/calendar.less

```diff
@@ -0,0 +1,408 @@
+.calendar_material {
+	position: absolute;
+	display: block;
+	background-color: white;
+	font-size: 14 * @px_base;
+	.font(regular);
+	color: #404040;
```

Comment:
What color value is # 404040? Is this the value defined in the GUI?

Reply:
I removed color. It is not necessary value.